### ISSUE TITLE
Select - Axe Violation - Invalid ARIA attribute values

### ISSUE THREAD

**Issue by sagemihir** [Sage/carbon#5467]
### Current behaviour
```

Impact:
critical
Found on:
05/09/2022 at 2:16 PM
Found:
Automatically
Issue Tags:
cat.aria
wcag2a
wcag412
Issue Description
Ensures all ARIA attributes have valid values

Element location
#RegisteredCountry
Element source
<input aria-invalid="false" autocomplete="off" aria-expanded="false" role="combobox" name="RegisteredCountry" type="text" id="RegisteredCountry" data-element="input" class="sc-dREXXX cgNdcz" value="Scotland" aria-activedescendant="edaa41e7-f70f-0b19-4805-5e4b24e40f31">
To solve this issue, you need to...
Fix the following:
Invalid ARIA attribute value: aria-activedescendant="edaa41e7-f70f-0b19-4805-5e4b24e40f31"

skip to
main content
```

### Expected behaviour

No accessibility violations should occur.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/select--default-story

### JIRA Ticket (Sage Only)

AC-7291  

### Carbon Version

110.6.0

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

MacOS

### Anything else we should know?

Related issue - https://github.com/Sage/carbon/issues/5289

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by DipperTheDan** [comment]
FE-5363

**Comment by DipperTheDan** [comment]
This issue seems to be a duplicate of #5289 which has since been closed and a fix released in v110.4.1.
